pkgs/development/tools/godot/3/debug-server.nix

```diff
@@ -0,0 +1,7 @@
+{ godot-headless }:
+
+godot-headless.overrideAttrs (self: base: {
+  pname = "godot-debug-server";
+  buildDescription = "debug server";
```

Comment:
```suggestion
  buildDescription = "godot debug server";
```

Reply:
This `buildDescription` attribute exists only in the godot derivations, and it is plugged into the `meta.description` standard attribute.  In that context, it's already clear that we're talking about godot, so adding the word "godot" to this would be redundant.